.github/workflows/playwright.yml

```diff
@@ -0,0 +1,23 @@
+name: Playwright Tests
+on: [push, pull_request]
+jobs:
+  test:
+    timeout-minutes: 5
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - uses: actions/setup-node@v3
+      with:
+        node-version: 18
+    - name: Install dependencies
+      run: npm ci
+    - name: Install Playwright Browsers
+      run: npx playwright install --with-deps
+    - name: Run Playwright tests
+      run: npx playwright test
+    - uses: actions/upload-artifact@v3
+      if: always()
+      with:
+        name: playwright-report
+        path: playwright-report/
+        retention-days: 30
```

Comment:
what is retention-days 30 mean? (copied from old PR) @AjayBenno 

https://github.com/yext/analytics/pull/78/files/43fe152ffcf9a9768f99d1c39b764777f6805c56#r1326142207

Reply:
I think it means how long the artifact produced by the run is retained. I'm not seeing anything online that would give me an exact definition. If this is the case, 30 days is definitely overkill. Thoughts on 2 or 3 days?

Reply:
Apparently it doesn't really cost us anymore so 30 seems fine for now.